java/hibernate-ucp/src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN" 
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+    <session-factory>
+        <property name="hbm2ddl.auto">create</property>
+        <property name="connection.url">url</property>
+        <property name="connection.username">user</property>
+        <property name="connection.password">password</property>
+        <property name="connection.driver_class">oracle.jdbc.driver.OracleDriver</property>
```

Comment:
Why do you have both connection.driver_class and oracleucp.connectionFactoryClassName? It looks redundant.

Reply:
driver_class is mandatory for hibernate